src/Mailgun/Resource/Api/Domains/DeliverySettingsResponse.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Domains;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class DeliverySettingsResponse
+{
+    /**
+     * @var string
+     */
+    private $noVerify;
+
+    /**
+     * @var string
+     */
+    private $requireTLS;
+
+    public function __construct($data)
+    {
+        $connSettings = $data['connection'];
+        $this->noVerify = $connSettings['skip_verification'] ? 'true' : 'false';
+        $this->requireTLS = $connSettings['require_tls'] ? 'true' : 'false';
```

Comment:
You're assuming the expected key (`$data['connection']`) exists without validation. Then you are just checking if a sub-key exists to set a predefined value.

Instead of this, I would like to suggest to set 2 arguments on this constructor, and set the values directly (don't forget to validate them, to ensure we get valid values).